packages/cubejs-elasticsearch-driver/driver/ElasticSearchDriver.js

```diff
@@ -36,14 +52,13 @@ class ElasticSearchDriver extends BaseDriver {
       })).body;
 
       // INFO: cloud left in place for backward compatibility
-      if (this.config.cloud || this.config.queryFormat === 'jdbc') {
+      if (this.config.cloud || (['jdbc', 'json'].indexOf(this.config.queryFormat) > -1)) {
```

Comment:
I think it's better to use `includes` instead of `indexOf`

https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
thanks, done